web/src/LogPane.tsx

```diff
@@ -211,31 +247,26 @@ class LogPane extends Component<LogPaneProps, LogPaneState> {
       }
 
       let el = (
-        <span
+        <LogLineComponent
           ref={i === 0 ? this.highlightRef : null}
```

Comment:
@jazzdan do you understand what this `i == 0` check is trying to accomplish? i spent some time poking at it and couldn't figure it out. it seems wrong

Reply:
I agree, that seems wrong.